game.cpp

```diff
@@ -1210,8 +1210,6 @@ void test_game() //!OCLINT tests may be many
   }
 #endif
 
-  //#define FIX_ISSUE_340
-#ifdef FIX_ISSUE_340
   // make sure that eat_nth_food() throws a logic_error when the food is already eaten
   {
```

Comment:
since this it the test the issue is required to operate.
The issue itself should be created and defined correctly, marked with [WIP]
this is needed to validate that the test is confirming the needed behavoir